micro-rdk/src/common/grpc.rs

```diff
@@ -32,7 +32,7 @@ use std::rc::Rc;
 use std::task::{Context, Poll};
 use thiserror::Error;
 
-use super::webrtc::grpc::WebRtcGrpcService;
+use super::{motor::Motor, webrtc::grpc::WebRtcGrpcService};
```

Comment:
nit: move this up to the imports at L10. Could probably move the `webrtc` import there too if no one else minds a little code movement.